package.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<package format="3">
+  <name>jrl-cmakemodules</name>
+  <version>0.0.0</version>
+  <description>CMake utility toolbox</description>
+
+  <maintainer email="justin.carpentier@inria.fr">Justin Carpentier</maintainer>
+  <maintainer email="gsaurel@laas.fr">Guilhem Saurel</maintainer>
+  <maintainer email="fill.me@in.com">Pierre Gergondet</maintainer>
```

Comment:
```suggestion
  <maintainer>Pierre Gergondet</maintainer>
```

Reply:
Unfortunately I cannot as the xml note maintainer asks for an email.

Reply:
I guess this is because this is mostly for use by ROS buildfarm, to spam people responsible for broken packages. I think nobody want that. Can we use an "author" node instead, then ?

Reply:
Hi I need I maintainer in the package.xml and I can put the rest as authors.
Can I put you @nim65s as sole maintainer?
And the rest as author?
So there is on email that is being spammed?

Reply:
yes